applications/project_aurras_mvp_phase_2.md

```diff
@@ -237,3 +234,16 @@ We came across the Web3 Grants Program while perusing about the developments in
 Here you can also add any additional information that you think is relevant to this application but isn't part of it already, such as:
 
 - Previous grant: https://github.com/w3f/Grants-Program/pull/163
+
+## Update & Amendments
+
+### 28/12/2022
+
+We have encountered a critical issue where we faced workflows fails due to certain Openwhisk actions timing out hence we took an alternative approach where we have made Polkadot API wasm compatible and included as a plugin to the wasm, due to this the timeline has now been exceeded by several months and has made certain components we developed obsolete.
+
+| Number | Deliverable | Specification | Link |
+| ------------- | ------------- | ------------- | ------------- |
+| 0a. | staking and payout features for [scs/substrate-api-client](https://github.com/scs/substrate-api-client) | staking functionalities and payout example. fixed bugs in staking modules | https://github.com/scs/substrate-api-client/pull/294 |
+
+- **Estimated Duration:** 45 Working days
+- **FTE:**  1
```

Comment:
What is this and how is it related to the rest of the project? If you want to deliver it with the next milestone, could you add it to its deliverable table?

Reply:
We have used [scs/substrate-api-client](https://github.com/scs/substrate-api-client) as a wasm-compatible lib for the workflow to interact with substrate-based chains. We have made some contributions that would be needed for our use case. We will add this to the deliverables. Thank you